docs/reference/announcements-release-notes/overview.md

```diff
@@ -0,0 +1,65 @@
+---
+id: overview
+title: "Announcements & release notes"
+description: "Important announcements and release notes for upcoming and past Camunda 8 releases that customers should be aware of."
+---
+
+In this section you will find announcements and release notes for each supported version of the Camunda documentation.
+
+**Announcements** include:
+
+- Deprecations and breaking changes
+- Supported environments (updates regarding OpenJDK, Spring Boot, Elasticsearch, Keycloak, Amazon OpenSearch, Docker tags, Helm chart versions, etc.)
+
+:::note
+
+Updating your Camunda 8 application or server installation from one Camunda 8 version to the other with Self-Managed? Visit our [update guides](/self-managed/operational-guides/update-guide/introduction.md).
+
+The [supported environments page](/reference/supported-environments.md) additionally lists browsers, operating systems, clients, deployment options, and component requirements, which are tested and supported for compatibility with Camunda 8.
+:::
+
+**Release notes** include:
+
+- New product features and enhancements
+- Bug fixes
+
```

Comment:
Is this true? Bug fixes are really only in the CHANGELOGs IM

```suggestion
- Key new product features and enhancements

```

Reply:
Adjusting in incoming commit.